docs/guide/multisite/multisite_configuration.md

```diff
@@ -158,6 +158,12 @@ ezplatform:
                     - another_domain.org
 ```
 
+!!! caution "SiteAccess with separate admin domain"
+
+    If an admin SiteAccess in your installation uses a different domain than the front SiteAccesses,
+    be sure to use SSL (https protocol).
```

Comment:
I don't see why the fix for https://github.com/ezsystems/developer-documentation/pull/1431 has any realtion to https vs http..? This statement is simply false, or am I missing anything @webhdx (https://github.com/webhdx)

What is really important is that trusted_proxies are configured correctly if you have varnish, fastly or any HTTPS termination in front of the webserver..

Reply:
I don't exactly remember what was to reason to require SSL but it may be related to the frontend code.

Reply:
Okay. I tested both with HTTP and with HTTPS and I got both working.